package.json

```diff
@@ -27,6 +27,7 @@
     "docs": "npm run docs:lint && npm run docs:api",
     "jsdoc": "jsdoc",
     "docs:api": "jsdoc -c .jsdoc.json",
+    "postdocs:api": "node ./build/fix-api-docs.js",
```

Comment:
this runs immediately after `docs:api` and before the `prepublish` script finishes, as an example?